package.json

```diff
@@ -2,22 +2,49 @@
   "name": "@mia-platform/eslint-config-mia",
   "version": "1.7.9",
   "description": "Mia-Platform javascript lint rules",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo no test available"
+  "keyword": [
+    "Mia-Platform JS Style",
+    "code linter",
+    "code style",
+    "eslint",
+    "eslintconfig",
+    "es2018",
+    "javascript",
+    "lint",
+    "policy",
+    "quality",
+    "styleguide",
+    "style linter"
+  ],
+  "homepage": "https://www.mia-platform.eu/",
+  "bugs": {
+    "url": "https://github.com/mia-platform/eslint-config-mia/issues",
+    "email": "core@mia-platform.eu"
   },
-  "author": "mia-platform",
   "license": "MIT",
-  "private": false,
-  "peerDependencies": {
-    "eslint": ">= 3"
-  },
+  "author": "Mia Platform Core Team<core@mia-platform.eu>",
```

Comment:
This is not compliant https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Reply:
Missed the space, easy fix.